pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -286,6 +287,89 @@ func (a *ofFlowAction) Group(id GroupIDType) FlowBuilder {
 	return a.builder
 }
 
+//  Learn is an action adds or modifies a flow in an OpenFlow table.
+func (a *ofFlowAction) Learn(id TableIDType, priority uint16, idleTimeout uint16, cookie uint64) LearnAction {
+	la := &ofLearnAction{
+		flowBuilder: a.builder,
+		nxLearn:     ofctrl.NewLearnAction(uint8(id), priority, idleTimeout, 0, 0, 0, cookie),
+	}
+	la.nxLearn.DeleteLearnedFlowsAfterDeletion()
```

Comment:
should this be an option? If not (because we don't need it for Antrea) maybe worth mentioning in the comment.